nrf_wifi/fw_if/umac_if/inc/fw/patch_info.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @brief nRF70 patch version.
+ */
+#ifndef _PATCH_VERSION_H
+#define _PATCH_VERSION_H
+
+#include "pack_def.h"
+
+#ifndef BIT
+#define BIT(x) (1 << (x))
+#endif
+
+/* 2 - LMAC and 2 - UMAC */
+#define NRF_WIFI_PATCH_NUM_IMAGES (2 + 2)
+
+enum nrf70_feature_flags {
+	NRF70_FEAT_SYSTEM_MODE = BIT(0),
+	NRF70_FEAT_RADIO_TEST = BIT(1),
+	NRF70_FEAT_SCAN_ONLY = BIT(2),
+};
+
+enum nrf70_image_ids {
+	NRF70_IMAGE_UMAC_PRI = 0,
+	NRF70_IMAGE_UMAC_SEC,
+	NRF70_IMAGE_LMAC_PRI,
+	NRF70_IMAGE_LMAC_SEC,
+};
+
+struct nrf70_fw_image {
+	unsigned int type;
+	unsigned int len;
+	/* Data follows */
```

Comment:
Is it better to have this as a 0 byte array?

Reply:
That isn't allowed in Zephyr as its non-standard, it has to be flexible array, but as we don't use that, just removed.